$OpenBSD: patch-src_typespeed_c,v 1.1.1.1 2016/01/22 15:24:30 landry Exp $

Disable the code opening the systemwide highscore file
and dropping privileges, since we're removing the setgid bit

--- src/typespeed.c.orig	Fri Jan 22 14:38:01 2016
+++ src/typespeed.c	Fri Jan 22 14:39:59 2016
@@ -318,6 +318,7 @@ main(int argc, char **argv)
 	else
 		progname++;
 
+#if 0
 	/* just open high score file while being setgid games */
 	if ((hfd = open(HIGHFILE, O_RDWR, 0)) == -1)
 		xerr(1, "main: open: %s", HIGHFILE);
@@ -351,6 +352,7 @@ main(int argc, char **argv)
 	if (hfd == STDIN_FILENO || hfd == STDOUT_FILENO ||
 	    hfd == STDERR_FILENO)
 		exit(1);
+#endif
 	if (!isatty(STDIN_FILENO) || !isatty(STDOUT_FILENO) ||
 	    !isatty(STDERR_FILENO))
 		xerrx(1, "not fully connected to a terminal");
